pages/sdk/api-kit.mdx

```diff
@@ -19,122 +20,132 @@ In this guide we will see how to propose transactions to the service and collect
-  to: '0x',
-  value: '1', // 1 wei
-  data: '0x',
-  operation: OperationType.Call
-}
+  const safeTransaction = await protocolKit.createTransaction({
+    transactions: [safeTransactionData]
+  })
 
-const safeTransaction = await protocolKit.createTransaction({ transactions: [safeTransactionData] })
+  const senderAddress = await signer.getAddress()
+  const safeTxHash = await protocolKit.getTransactionHash(safeTransaction)
+  const signature = await protocolKit.signHash(safeTxHash)
 
-const senderAddress = await signer.getAddress()
-const safeTxHash = await protocolKit.getTransactionHash(safeTransaction)
-const signature = await protocolKit.signHash(safeTxHash)
+  // Propose transaction to the service
+  await apiKit.proposeTransaction({
+    safeAddress: await protocolKit.getAddress(),
+    safeTransactionData: safeTransaction.data,
+    safeTxHash,
+    senderAddress,
+    senderSignature: signature.data
+  })
+  ```
 
-// Propose transaction to the service
-await apiKit.proposeTransaction({
-  safeAddress: await protocolKit.getAddress(),
-  safeTransactionData: safeTransaction.data,
-  safeTxHash,
-  senderAddress,
-  senderSignature: signature.data
-})
-```
+  ### Retrieve the pending transactions
 
-## Retrieve the pending transactions
+  Different methods in the API Kit are available to retrieve pending transactions depending on the situation. To retrieve a transaction given the Safe transaction hash use the method that's not commented.
 
-Different methods in the API Kit are available to retrieve pending transactions depending on the situation. To retrieve a transaction given the Safe transaction hash use the method that's not commented.
+  ```typescript
+  const transaction = await service.getTransaction("<SAFE_TX_HASH>")
+  // const transactions = await service.getPendingTransactions()
+  // const transactions = await service.getIncomingTransactions()
+  // const transactions = await service.getMultisigTransactions()
+  // const transactions = await service.getModuleTransactions()
+  // const transactions = await service.getAllTransactions()
+  ```
 
-```typescript
-const transaction = await service.getTransaction("<SAFE_TX_HASH>")
-// const transactions = await service.getPendingTransactions()
-// const transactions = await service.getIncomingTransactions()
-// const transactions = await service.getMultisigTransactions()
-// const transactions = await service.getModuleTransactions()
-// const transactions = await service.getAllTransactions()
-```
+  ### Confirm the transaction
 
-## Confirm the transaction
+  In this step we need to sign the transaction with the Protocol Kit and submit the signature to the Safe Transaction Service using the `confirmTransaction` method.
 
-In this step we need to sign the transaction with the Protocol Kit and submit the signature to the Safe Transaction Service using the `confirmTransaction` method.
+  ```typescript
+  const safeTxHash = transaction.transactionHash
```

Comment:
We should include a `connect()` line here because you need to sign with another owner

Reply:
Correct. Will create a new PR, as I was already planning to fix somethin else in the API Kit guide